scrap/main.py

```
@@ -0,0 +1,52 @@
+        title = item.get("title", "")
+        author = item.get("author", "")
+        publisher = item.get("publisher", "")
+        description = item.get("description", "")
+        image_url = item.get("image_url", "")
+        isbn = item.get("isbn", "")
+        raw_date = item.get("published_date", "")
+
+        published_date = None
+        if raw_date and len(raw_date) == 8:
+            try:
+                dt = datetime.strptime(raw_date, "%Y%m%d")
```

Ruff (0.11.9)

Naive datetime constructed using `datetime.datetime.strptime()` without %z

(DTZ007)